app/src/main/java/net/gsantner/markor/web/MarkorWebViewClient.java

```diff
@@ -62,4 +79,20 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
         }
         return true;
     }
+
+    @Override
+    public void onPageStarted(final WebView view, final String url, final Bitmap favicon) {
```

Comment:
For some added safety I added this to disable file access for non-local files.

Most of this PR is to be looked at as an experiment, not to merged anytime soon :)